wf-core-tck-runner/pom.xml

```diff
@@ -90,7 +90,7 @@
         <jsonp.api.version>2.1.0</jsonp.api.version>
         <jsonb.api.version>3.0.0</jsonb.api.version>
         <rest.api.version>3.1.0</rest.api.version>
-        <core.profile.tck.version>10.0.0</core.profile.tck.version>
```

Comment:
Since https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-core-profile-tck-10.0.1.zip is not yet promoted to https://download.eclipse.org/jakartaee/coreprofile/10.0/jakarta-core-profile-tck-10.0.1.zip, would users override `core.profile.tck.version` to be `10.0.0` until the staged TCK ^ is promoted as final?

Reply:
Yes, that would work. However, I definitely see the 10.0.1 download there https://download.eclipse.org/jakartaee/coreprofile/10.0/